scripts/jenkins/jobs-ibs/cloud-mkcloud6-job-hyperv.yaml

```diff
@@ -0,0 +1,25 @@
+- job:
+    name: 'cloud-mkcloud6-job-hyperv'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H H * * *'
```

Comment:
the plan behind is that jenkins choses a random minute and hour, so that when copying a job, not all of them run at the same time. At least for the minute part we used that everywhere.